conan/cli/commands/audit.py

```diff
@@ -20,15 +20,22 @@ def _add_provider_arg(subparser):
     subparser.add_argument("-p", "--provider", help="Provider to use for scanning")
 
 
-def _parse_error_threshold(result, error_level):
-    """Mark the result as error if any of the vulnerabilities has a severity greater than or equal to"""
+def _parse_error_threshold(result: dict, error_level: float):
```

Comment:
How can the ``type`` be ``float``, then in code do ``if error_level is not None``, and ``float(error_level)``.
I'd say better remove the type (otherwise it should be kind of an ``Optional[float]``?

Reply:
Reverted by the commit 46aa6c3